modules/ROOT/pages/govcloud-data-control.adoc

```diff
@@ -0,0 +1,98 @@
+= GovCloud Data Control
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:imagesdir: ../assets/images/
+
+MuleSoft Government Cloud (GovCloud) organizations can restrict the data entities exposed to their Salesforce Community Cloud ACM instance.
+
+Restrictions are defined on the Anypoint organization management plane and applied at the data bridge level.
+
+Before defining the entities and restrictions, get an authorization token from the Anypoint identity services API endpoint `/accounts/login` by sending the username and password of a user with the `Organization Administrator` permission.
```

Comment:
```suggestion
Before defining the entities and restrictions, get an authorization token from the Anypoint identity services API endpoint `/accounts/login` by sending the username and password of a user with the Organization Administrator permission.
```

Reply:
From this point onward, I will restrict comments per your request for a CE-level review.
However, I think that this text needs to be reviewed for verbosity and choppy narrative flow at some point in the future.

Reply:
Fixed.